tests/integration/cmd_url_test.go

```diff
@@ -88,6 +88,21 @@ var _ = Describe("odo url command tests", func() {
 		})
 	})
 
+	Context("Describing urls", func() {
+		It("should describe appropriate URLs and push message", func() {
```

Comment:
please add some more cases with the various pushed states or maybe even add them here itself

Reply:
ok